websites/T/TwitCasting/presence.ts

```diff
@@ -0,0 +1,192 @@
+let elapsed = Math.floor(Date.now() / 1000),
+	oldLang = "en",
+	prevUrl = document.location.href;
+
+const presence = new Presence({
+		clientId: "951488835718635600"
+	}),
+	getStrings = async () => {
+		return presence.getStrings(
+			{
+				play: "general.playing",
+				pause: "general.paused",
+				live: "general.live",
+				viewHome: "general.viewHome",
+				viewProfile: "general.viewProfile",
+				buttonViewProfile: "general.buttonViewProfile",
+				browse: "general.browsing",
+				watchingLive: "general.watchingLive",
+				watchingVid: "general.watchingVid",
+				viewCategory: "general.viewCategory",
+				viewTheir: "twitch.viewTheir",
+				followList: "twitch.followList",
+				dashboardManage: "twitch.dashboardManage",
+				subs: "twitch.subs",
+				browsingVideos: "youtube.browsingVideos",
+				ofChannel: "youtube.ofChannel",
+				searchingFor: "general.searchFor",
+				searchingSomething: "general.searchSomething",
+				search: "general.search",
+				watchStream: "general.buttonWatchStream",
+				watchVideo: "general.buttonWatchVideo"
```

Comment:
Best for consistency
```suggestion
				playing: "general.playing",
				paused: "general.paused",
				live: "general.live",
				viewHome: "general.viewHome",
				viewProfile: "general.viewProfile",
				buttonViewProfile: "general.buttonViewProfile",
				browsing: "general.browsing",
				watchingLive: "general.watchingLive",
				watchingVid: "general.watchingVid",
				viewCategory: "general.viewCategory",
				viewTheir: "twitch.viewTheir",
				followList: "twitch.followList",
				dashboardManage: "twitch.dashboardManage",
				subs: "twitch.subs",
				browsingVideos: "youtube.browsingVideos",
				ofChannel: "youtube.ofChannel",
				searchFor: "general.searchFor",
				searchSomething: "general.searchSomething",
				search: "general.search",
				buttonWatchStream: "general.buttonWatchStream",
				buttonWatchVideo: "general.buttonWatchVideo"
```

Reply:
Done